docs/content/middlewares/http/errorpages.md

```diff
@@ -112,6 +113,11 @@ The service that will serve the new requested error page.
 
     In Kubernetes, you need to reference a Kubernetes Service instead of a Traefik service.
 
+!!! info "Host Header"
+
+    By default, the client `Host` header value is forwarded to the configured error [service](#service).
+    To forward the `Host` value corresponding to the configured error service URL, the [passHostHeader](../../../routing/services/#pass-host-header) option must be used.     
```

Comment:
Would it be worth calling out that this option needs to be set to `false` if you wish to have the host header inferred?

Reply:
```suggestion
    To forward the `Host` value corresponding to the configured error service URL, the [passHostHeader](../../../routing/services/#pass-host-header) option must be set to `false`.     
```